samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -9,13 +9,20 @@
 
 from samcli.cli.cli_config_file import configuration_option, TomlProvider
 from samcli.cli.main import pass_context, common_options, aws_creds_options, print_cmdline_args
+
+from samcli.commands.pipeline.bootstrap.pipeline_oidc_provider import PipelineOidcProvider
 from samcli.lib.config.samconfig import SamConfig
-from samcli.lib.pipeline.bootstrap.stage import Stage
+
+from samcli.commands.pipeline.bootstrap.oidc_config import (
+    BitbucketOidcConfig,
+    GitHubOidcConfig,
+    OidcConfig,
+    GitLabOidcConfig,
+)
+
```

Comment:
Apologies for been picky, but these imports can be moved down as well.

Reply:
These imports are also used in another function as input, so I think they need to be at the top. Correct me if I'm wrong.

Reply:
looks like they are used in other functions only for type indication. We can wrap them by quotes instead, e.g.
```
def my_function(arg: "MyClass"):
    pass
```

Reply:
Good suggestion! Make it work. Thanks!